### ISSUE TITLE
MNIST Conv net doesn't converge

### ISSUE THREAD

**Issue by tim-impactia** [m4nuC/aifiddle-issues#10]
Hello,

Using AI fiddle to create fully connected nets and single layer conv nets converges OK, however none of my attempts to use 2-3 convolution layers seem to get much above 50% accuracy.

To get a reference point I re implemented in AI Fiddle the simple conv net here: https://github.com/keras-team/keras/blob/master/examples/mnist_cnn.py and also ran it in a colaboratory notebook. The notebook was at 92% accuracy after the first epoch. In AIFiddle I get to around 50% with 1 epoch and a batch size of 32. Moving to a batch size of 128 (as used by the code sample) sees this crashing down to 10%.

Do you have any idea what could cause this?

**Comment by m4nuC** [comment]
Hi Tim, 

Apologies for the delay, i've missed the notification.

Did you by chance save your experiment in AIFIddle ? If so, could you share that link ? 

I've made many of such architecture in AiFIddle and it should work fine.

Thanks

**Comment by tim-impactia** [comment]
Hi Emmanuel ,

Hello I saved it on the Cloud with the name "Hushed Hammer", can you see it?

Thank you for creating aifiddle by the way, it's fantastic for teaching basic principles and experimenting.

**Comment by tim-impactia** [comment]
In case this helps: during a session where I showed aifiddle to 3 other devs we recreated that exact network. For 2 of us the network converged (> 90% accuracy), for the other 2 it got stuck around 50%. We couldn't see any differences that would explain it (I remained one of those for whom the network was stuck at 50%).

**Comment by m4nuC** [comment]
Yes that one: https://editor.aifiddle.io/models/cloud/831bcf2c-76f6-43ba-97f1-26ff4ec2f1e2. Thanks for the added info I will investigate. I am cooking up a major release now so that should come with that

**Comment by m4nuC** [comment]
>In case this helps: during a session where I showed aifiddle to 3 other devs we recreated that exact network. For 2 of us the network converged (> 90% accuracy), for the other 2 it got stuck around 50%. We couldn't see any differences that would explain it (I remained one of those for whom the network was stuck at 50%).

Would you by any chance have a link to those models ?
